buf.work.yaml

```diff
@@ -0,0 +1,3 @@
+version: v1
```

Comment:
should we move to v2 or is it ok to stick with v1?

Reply:
version indicates configuration format, v1 is still supported. should be ok to stick with it and migrate later